### PR TITLE
[IMP] sales: updated deadline doc for 16

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3460706&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/sales/sales/send_quotations/deadline.rst [hunk 1]
```diff
@@ -7,54 +7,50 @@ encourages customers to act quickly during sales negotiations, for they might fe
...
 .. image:: deadline/quotation-deadlines-expiration-field.png
    :align: center
-   :alt: How to configure deadlines on Odoo Sales.
+   :alt: The expiration field on a standard quotation form in Odoo Sales.
 
-Once the expiration date is selected, click :guilabel:`Save` to save all changes.
+.. tip::
+   By clicking the :guilabel:`Preview` button on a quotation, Odoo clearly displays when that
+   specific offer expires. As a reminder, the number of days is the same as those mentioned in the
+   quotation template (if a quotation template was used for the initial quotation).
```
Comment (jero-odoo) [inline content/applications/sales/sales/send_quotations/deadline.rst:28]:
```suggestion
   specific offer expires. As a reminder, the expiration date is the same as those mentioned in the
   quotation template (if a quotation template was used for the initial quotation).
```
I know the quotation template lets you set the expiration date out by a specific number of days, but I was only able to get the preview to show the "This offer expires on" field with the expiration date, and not in a "in X amount of days". I am not sure if I am doing something wrong (I thought you could have it read by amount of days but I could be mistaken)

File: content/applications/sales/sales/send_quotations/deadline.rst [hunk 2]
```diff
@@ -7,54 +7,50 @@ encourages customers to act quickly during sales negotiations, for they might fe
...
+Quotation template expiration
+=============================
 
-The Odoo *Sales* application also makes it possible to add a deadline to every quotation template.
-Whenever a specific quotation template is used in a quote, its associated deadline is automatically
-applied.
+The Odoo *Sales* application also makes it possible to add a deadline expiration date to quotation
+templates.
 
 To add a deadline to a quotation template, navigate to :menuselection:`Sales app --> Configuration
```
Comment (jero-odoo) [inline content/applications/sales/sales/send_quotations/deadline.rst:40]:
```suggestion
To add a deadline expiration date to a quotation template, navigate to :menuselection:`Sales app --> Configuration
```

File: content/applications/sales/sales/send_quotations/deadline.rst [hunk 3]
```diff
@@ -7,54 +7,50 @@ encourages customers to act quickly during sales negotiations, for they might fe
...
-.. tip::
-   By clicking the :guilabel:`Customer Preview` button on a quotation, Odoo clearly displays when
-   that specific offer expires. As a reminder, the number of days is the same as those mentioned in
-   the quotation template (if a quotation template was used for the initial quotation).
-
-   .. image:: deadline/quotation-deadlines-preview.png
-      :align: center
-      :alt: How customers will see deadlines on Odoo Sales.
+Then, whenever that specific quotation template is used in a quote, its associated deadline is
+automatically applied.
```
Comment (jero-odoo) [inline content/applications/sales/sales/send_quotations/deadline.rst:53]:
```suggestion
Then, whenever that specific quotation template is used in a quote, an expiration date is automatically calculated based on the number of days designated above. However, this date can be overwritten before sending the quotation to the customer.
```

**Review by jero-odoo — APPROVED** [review]
@ksc-odoo Nice work, just a few notes, otherwise ready to go 👍

File: content/applications/sales/sales/send_quotations/deadline.rst (hunk 1 above)
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/deadline.rst:28]: I just double-checked, and you are totally right. It only shows the specific date, *not* the number of days. Thanks for catching this!

**Comment by ksc-odoo** [comment]
Thanks for the super-helpful feedback, @jero-odoo ! I made all your suggested edits, and I'm passing this off to the Materials team for a 2nd review! @hojo-odoo @Felicious @brse-odoo

**Review by hojo-odoo — APPROVED** [review]
Hey @ksc-odoo,

Great job on this doc! I didn't notice any problems, so feel free to tag it for final review.

Thanks,
hojo

**Comment by StraubCreative** [comment]
@samueljlieber this one's good for tech review please 🙏

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo thanks for this PR! Your improvements look good to me 🙂 Approving now and passing to @StraubCreative!

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**